packages/esm-patient-clinical-view-app/src/clinical-view-overview/clinical-view-overview.component.tsx

```diff
@@ -0,0 +1,99 @@
+import React from "react";
+import Button from "carbon-components-react/es/components/Button";
+import Tab from "carbon-components-react/es/components/Tab";
+import Tabs from "carbon-components-react/es/components/Tabs";
+import { useTranslation } from "react-i18next";
+import {
+  attach,
+  ExtensionSlot,
+  useConfig,
+  useCurrentPatient,
+} from "@openmrs/esm-framework";
+import tail from "lodash-es/tail";
+import EmptyDataIllustration from "../empty-state/empty-data-illustration.component";
+import styles from "./clinical-view-overview.component.scss";
+
+const ClinicalViewOverview: React.FC = () => {
+  const config = useConfig();
+  const { t } = useTranslation();
+  const [, patient, patientUuid] = useCurrentPatient();
```

Comment:
We should not use `useCurrentPatient`. Usually, you should obtain `patient` and `patientUuid` from the props. How is the `ClinicalViewOverview` used?

Reply:
(It seems its exclusively used as a dashboard. Dashboards get these props).